### ISSUE TITLE
Is possible to share your pre-train model?

### ISSUE THREAD

**Issue by Leeviber** [okankop/ASDNet#3]
Hi author,

Could you publish your model which obtain the performance of the paper, Thanks!

**Comment by robotNo201** [comment]
i dont think so, the lab guys just wanna get their things down, most github projects are not reproducible, u know that, but they need code to convince their submitted paper is correct, the best way will be to contact the publisher that their code not working.
